base/libgit2/rebase.jl

```diff
@@ -16,6 +16,14 @@ function Base.count(rb::GitRebase)
     return ccall((:git_rebase_operation_entrycount, :libgit2), Csize_t, (Ptr{Void},), rb.ptr)
 end
 
+"""
+    current(rb::GitRebase) -> Csize_t
+
+Return the index of the current [`RebaseOperation`](@ref). If no operation has
+yet been applied (because the [`GitRebase`](@ref) has been constructed but `next`
+has not yet been called or iteration over `rb` has not yet begun), return
+`GIT_REBASE_NO_OPERATION`, which is equal to `SIZE_MAX` of `Csize_t`.
```

Comment:
for someone who knows julia but not C, SIZE_MAX isn't very meaningful - even if you do know C, it's the kind of thing I'd need to look up every time